.github/workflows/conformance.yml

```diff
@@ -66,8 +66,8 @@ jobs:
       env:
         CC: gcc-9
         CXX: g++-9
-        HUNTER_UPLOAD_USERNAME: ${{ secrets.HUNTER_UPLOAD_USER }}
-        HUNTER_UPLOAD_PASSWORD: ${{ secrets.HUNTER_UPLOAD_TOKEN }}
+        GITHUB_HUNTER_USERNAME: ${{ secrets.HUNTER_UPLOAD_USER }}
+        GITHUB_HUNTER_TOKEN: ${{ secrets.HUNTER_UPLOAD_TOKEN }}
```

Comment:
Not sure why you would change that. Our CMake code uses these variable to upload to our binary cache. Please do not change those.

Reply:
I see. Then we need either both, or disable hunter upload for kagome. I'll try first.

Reply:
The upload works just fine as it is, to w3f own cache with these settings. If you enable kagome's own upload too, it is not only redundant, but also to a cache we do not control or access too.